frontend/apps/marketing/src/components/contentful/image/Image.tsx

```diff
@@ -1,47 +1,81 @@
-import React, {useMemo} from 'react';
-
-import DSCOImage from '@code-dot-org/component-library/image';
+'use client';
```

Comment:
This is needed in order to create custom components that act like built-in components so we can style them across multiple sites. (See [docs](https://mui.com/material-ui/customization/creating-themed-components/))

Reply:
I think this is fine for now